### PR TITLE
Added auth ,dmesg, dpkg/yum log files in supportBundle

### PR DESCRIPTION
**Pull request by ShaunakJoshi1407**
- #### Added platform-specific auth ,dmesg ,dpkg/yum log files in supportBundle as per https://www.eurovps.com/blog/important-linux-log-files-you-must-be-monitoring/
 
- ####  Made small changes in Remote execution in supportBundle
- #### Removed unused code from executor.go 

**Ubuntu**

Screenshot 2021-11-18 at 11 37 07 AM
)

SupportBundle contents:
Screenshot 2021-11-17 at 3 19 42 PM

**Centos**

Screenshot 2021-11-18 at 11 39 55 AM

SupportBundle contents:

Screenshot 2021-11-17 at 3 18 08 PM

### CODE REVIEW

File: pkg/supportBundle/supportBundle.go [hunk 1]
```diff
@@ -230,3 +275,8 @@ func GenSupportBundle(exec cmdexec.Executor, timestamp time.Time) (string, error
 	}
 
 }
+
+//To assign platform-specific auth,messages,dpkg/yum files
+func GenFiles(os string, exec cmdexec.Executor) {
+
+}
```
Comment (anupbarve) [inline pkg/supportBundle/supportBundle.go:282]: Remove ?

File: pkg/supportBundle/supportBundle.go [hunk 2]
```diff
@@ -187,6 +199,39 @@ func GenSupportBundle(exec cmdexec.Executor, timestamp time.Time) (string, error
...
 	}
 
+	_, errDmesg := exec.RunWithStdout("bash", "-c", fmt.Sprintf("stat %s", util.DmesgLog))
+	if errVar != nil {
+		zap.S().Debug("Dmesg files not Found!! ", errDmesg)
+	}
+
+	//Assign specific files according to the platform
+	//GenFiles(os, exec)
+
```
Comment (anupbarve) [inline pkg/supportBundle/supportBundle.go:209]: remove ?

File: cmd/checkNode.go [hunk 3]
```diff
@@ -52,7 +52,7 @@ func checkNodeRun(cmd *cobra.Command, args []string) {
 	detachedMode := cmd.Flags().Changed("no-prompt")
 	isRemote := cmdexec.CheckRemote(nc)
 
-	if cmdexec.CheckRemote(nc) {
+	if isRemote {
```
Comment (anupbarve) [inline cmd/checkNode.go:55]: is this a consequence of automation changes?

**Review by anupbarve — CHANGES_REQUESTED** [review]

File: pkg/supportBundle/supportBundle.go (hunk 1 above)
Comment (ShaunakJoshi1407) [reply pkg/supportBundle/supportBundle.go:282]: Yeah I'll remove this

File: pkg/supportBundle/supportBundle.go (hunk 2 above)
Comment (ShaunakJoshi1407) [reply pkg/supportBundle/supportBundle.go:209]: Yeah will remove this too

File: cmd/checkNode.go (hunk 3 above)
Comment (ShaunakJoshi1407) [reply cmd/checkNode.go:55]: Yeah....isRemote variable was declared and not used so I made this change

**Review by anupbarve — APPROVED** [review]

**Pull request merged**